src/node_http_parser.cc

```diff
@@ -959,6 +957,11 @@ void Initialize(Local<Object> target,
   target->Set(env->context(),
               FIXED_ONE_BYTE_STRING(env->isolate(), "HTTPParser"),
               t->GetFunction(env->context()).ToLocalChecked()).FromJust();
+
+#ifndef NODE_EXPERIMENTAL_HTTP
+  http_parser_set_max_header_size(
+    static_cast<uint32_t>(per_process_opts->max_http_header_size));
```

Comment:
Can we guard this with `uv_once()`? This module might be loaded more than once in the presence of Workers or embedder code